numpy/random/tests/test_extending.py

```diff
@@ -43,8 +43,8 @@
 @pytest.mark.skipif(sys.platform == 'win32', reason="cmd too long on CI")
```

Comment:
Do we still need to skip the test on windows?

Reply:
I'd like to just close the backport, it should be combined with this PR.

Reply:
Should be harmless to merge this one, and then close / reopen the backport

Reply:
Seems overly complicated for a one line change. Could just edit the other PR.

Reply:
ok, lets do it that way